src/registrar/templates/admin/base_site.html

```diff
@@ -24,34 +24,57 @@
 {% block title %}{% if subtitle %}{{ subtitle }} | {% endif %}{{ title }} | {{ site_title|default:_('Django site admin') }}{% endblock %}
 
 {% block extrastyle %}{{ block.super }}
-<link rel="stylesheet" type="text/css" href="{% static "css/styles.css" %}" />
+<link rel="stylesheet" type="text/css" href="{% static 'css/styles.css' %}" />
```

Comment:
(Q) Why was this changed?

Reply:
My ide doesn't like double quotes nested inside double quotes (it's one of MY nitpicks)